ui.content/src/main/content/jcr_root/content/hclecomm/us/en/HomePage/.content.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```

Comment:
@DeepaliAgrwl  Why home page checkedin as  a part of logo component?

Reply:
I created it for testing. But yes this should not be a part of testing. I have removed it from this branch and checked in the code as well